docs/DEVELOPING.md

```diff
@@ -441,6 +441,97 @@ $ make ftest NAME=sharepoint
 
 ℹ️ The connector uses the Elastic [ingest attachment processor](https://www.elastic.co/guide/en/enterprise-search/current/ingest-pipelines.html) plugin for extracting file contents. The ingest attachment processor extracts files by using the Apache text extraction library Tika. Supported file types eligible for extraction can be found as `TIKA_SUPPORTED_FILETYPES` in [utils.py](../connectors/utils.py) file.
 
+## ServiceNow Connector
+
+The [Elastic ServiceNow connector](../connectors/sources/servicenow.py) is provided in the Elastic connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
+
+### Availability and prerequisites
+
+This connector is available as a **connector client** from the **Python connectors framework**. To use this connector, satisfy all [connector client requirements](https://www.elastic.co/guide/en/enterprise-search/master/build-connector.html).
+
+This connector is in **beta** and is subject to change. The design and code is less mature than official GA features and is being provided as-is with no warranties. Beta features are not subject to the support SLA of official GA features.
+
+### Usage
+
+To use this connector as a **connector client**, use the **Customized connector** workflow.
+
+For additional operations, see [Usage](https://www.elastic.co/guide/en/enterprise-search/master/connectors-usage.html).
+
+### Compatibility
+
+ServiceNow "Tokyo", "San Diego" & "Rome" versions are compatible with Elastic connector frameworks.
+
+### Configuration
+
+The following configuration fields need to be provided for setting up the connector:
+
+##### `url`
+
+The host url of the account for ServiceNow.
+
+##### `username`
+
+The username of the account for ServiceNow.
+
+##### `password`
+
+The password of the account to be used for the ServiceNow.
+
+##### `services`
+
+Comma-separated list of Services to fetch data from ServiceNow. If the value is `*`, the connector will fetch data from list of basic services provided by ServiceNow: `Users`, `Incidents`, `Request Items`, `Knowledge Articles`, `Change Requests`. Default value is `*`. Examples:
+
+  - `Incident, User`
+  - `*`
+
+##### `retry_count`
+
+The number of retry attempts after failed request to ServiceNow. Default value is `3`.
+
+##### `concurrent_downloads`
+
+The number of concurrent downloads for fetching the attachment content. This speeds up the content extraction of attachments. Defaults to `10`.
+
+### Documents and syncs
+
+The connector syncs the following ServiceNow object types: 
+- **Records**
+- **Attachments**
+
+### Sync rules
+
+- Content of files bigger than 10 MB won't be extracted.
+- Permissions are not synced. **All documents** indexed to an Elastic deployment will be visible to **all users with access** to that Elastic Deployment.
+- Filtering rules are not available in the present version. Currently filtering is controlled via ingest pipelines.
```

Comment:
```suggestion
- Advanced sync rules are not available in the present version. Currently filtering is controlled via ingest pipelines.
```

Basic rules are available to every connector from a technical point of view. cc @danajuratoni Do we want to enable basic sync rules for ServiceNow or is there a good reason to disable them?